src/Text/Pandoc/Writers/OpenDocument.hs

```diff
@@ -684,14 +684,18 @@ listLevelStyle i =
                       , ("fo:margin-left", indent <> "in")
                       ]
 
-tableStyle :: Int -> [(Char,Double)] -> Doc Text
-tableStyle num wcs =
+tableStyle :: Int -> [(Text,Text)] -> [(Char,Double)] -> Doc Text
+tableStyle num kvs wcs =
     let tableId        = "Table" <> tshow (num + 1)
+        tableAttrStyles []   = []
+        tableAttrStyles ((k,v):xs)
+          | k == "width" = [("style:rel-width",v)] ++ tableAttrStyles xs
+          | otherwise = tableAttrStyles xs
         table          = inTags True "style:style"
                          [("style:name", tableId)
                          ,("style:family", "table")] $
                          selfClosingTag "style:table-properties"
-                         [("table:align"    , "center")]
+                         ([("table:align", "center")] ++ (tableAttrStyles kvs))
```

Comment:
The first argument to the list concatenation is a singleton list, so this could be written using the `:` cons operator instead of list concatenation.

Reply:
and done.